src/bio.js

```diff
@@ -1,15 +1,19 @@
+//p tag
+let firstSection = document.createElement("p");
+firstSection.id = "bio";
+firstSection.className = "myBio";
+
+//append p tag to div
+let bioDiv = document.getElementById("bio-section");
+bioDiv.appendChild(firstSection).innerHTML;
+
 function processData(response) {
-    const bio = response.bio;
-    const bio2 = response.bio2;
-    let output = "";
-    for(var i = 0; i < bio.length; i++){
-      output += bio[i];
-    }
-    for(let j = 0;j < bio2.length; j++){
-      output += bio2[j];
-    }
-     document.getElementById("bio").innerHTML = output;
+  let wrapList = document.createElement("li");
+  let listArray = document.getElementById("bio");
+  for (let i = 0; i < response.length; i++) {
+    listArray.appendChild(wrapList).innerHTML +=
+      response[i].bio.join(" ") + response[i].bio2.join(" ");
```

Comment:
If I'm reading this correctly, this is defeating the purpose of having bio and bio2. All you're doing here is combining everything into one string.  The bio section is probably an extremely bad use of the API request/JSON file anyway - do you want to get rid of all of this and just implement this section as static content?
If not, we can get this working properly, but you're going to need nested loops

Reply:
Can you explain why it's a case of bad use of the API request/JSON file? More than likely, I won't be adding more content to this section in the future, so it could be static. However, I still would like to create the nested loops example for future reference.